src/views/CardStack.js

```diff
@@ -36,6 +38,8 @@ import TransitionConfigs from './TransitionConfigs';
 
 const emptyFunction = () => {};
 
+const windowSize = Dimensions.get('window');
```

Comment:
updated, and the test failure is irrelevant with this PR, seems it was introduced by the recent change on the `Init` key